packages/core/src/features/telemetry/renderer/telemetry-white-list-for-functions.injectable.ts

```diff
@@ -91,13 +92,20 @@ const extensions = [
 
 const externalActions = ["open-link-in-browser"];
 
-const uiInteraction = ["show-details"];
+const uiInteraction = [{
+  id: "show-details",
+  getParams: (selfLink: string) => {
+    return {
+      kind: selfLink ? parseKubeApi(selfLink).resource : "",
+    };
+  },
+}];
 
 const terminal = ["create-terminal-tab"];
 
 export type WhiteListItem =
   | string
-  | { id: string; getParams: (...args: unknown[]) => AppEvent["params"] };
+  | { id: string; getParams: (...args: any[]) => AppEvent["params"] };
```

Comment:
technically `unknown[]` is more correct since the `id` could be wrong. But that is probably of low concern here. There are other improvements we could make in other places in the future.